docs/Dockerfile

```diff
@@ -12,8 +12,8 @@ FROM registry.access.redhat.com/ubi8-minimal:latest
 # Update RPMs to hopefully aid with vulns that aren't yet patched in the base image. Also remove package management
 # utils because our own policies don't recommend having them in the image. For that we need to be root.
 RUN microdnf upgrade && \
-    microdnf install \
-        python39 && \
+    microdnf install --setopt=tsflags=nodocs --setopt=install_weak_deps=0 \
```

Comment:
This will keep python installation minimal.

```diff
Installing: libtirpc;1.1.4-8.el8;x86_64;ubi-8-baseos-rpms
Installing: libnsl2;1.2.0-2.20180605git4a062cf.el8;x86_64;ubi-8-baseos-rpms
Installing: python39-setuptools-wheel;50.3.2-4.module+el8.5.0+12204+54860423;noarch;ubi-8-appstream-rpms
Installing: python39-pip-wheel;20.2.4-7.module+el8.6.0+13003+6bb2c488;noarch;ubi-8-appstream-rpms
Installing: expat;2.2.5-10.el8;x86_64;ubi-8-baseos-rpms
Installing: gdbm-libs;1:1.18-2.el8;x86_64;ubi-8-baseos-rpms
Installing: python39;3.9.13-2.module+el8.7.0+17195+44752b34;x86_64;ubi-8-appstream-rpms
Installing: python39-libs;3.9.13-2.module+el8.7.0+17195+44752b34;x86_64;ubi-8-appstream-rpms
< Installing: python39-setuptools;50.3.2-4.module+el8.5.0+12204+54860423;noarch;ubi-8-appstream-rpms
< Installing: python39-pip;20.2.4-7.module+el8.6.0+13003+6bb2c488;noarch;ubi-8-appstream-rpms

```